nextflow.config

```diff
@@ -10,221 +10,271 @@
+        conda.enabled = false
+        docker.enabled = false
+        singularity.enabled = false
+        podman.enabled = false
+        shifter.enabled = false
+        charliecloud.enabled = false
     }
     wave {
-        apptainer.ociAutoPull   = true
+        apptainer.ociAutoPull = true
         singularity.ociAutoPull = true
-        wave.enabled            = true
-        wave.freeze             = true
-        wave.strategy           = 'conda,container'
+        wave.enabled = true
+        wave.freeze = true
+        wave.strategy = 'conda,container'
     }
     gitpod {
-        executor.name           = 'local'
-        executor.cpus           = 4
-        executor.memory         = 8.GB
+        executor.name = 'local'
+        executor.cpus = 4
+        executor.memory = 8.GB
         process {
             resourceLimits = [
                 memory: 8.GB,
-                cpus  : 4,
-                time  : 1.h
+                cpus: 4,
+                time: 1.h,
             ]
         }
     }
-    test                          { includeConfig 'conf/test_targeted.config' }
-    test_targeted                 { includeConfig 'conf/test_targeted.config' }
-    test_full                     { includeConfig 'conf/test_targeted_full.config' }
-    test_targeted_full            { includeConfig 'conf/test_targeted_full.config' }
-    test_umis                     { includeConfig 'conf/test_umis.config' }
-    test_screening_full           { includeConfig 'conf/test_screening_full.config' }
-    test_screening                { includeConfig 'conf/test_screening.config' }
-    test_screening_paired         { includeConfig 'conf/test_screening_paired.config' }
-    test_screening_rra            { includeConfig 'conf/test_screening_rra.config' }
-    test_screening_count_table    { includeConfig 'conf/test_screening_count_table.config' }
+    test {
+        includeConfig 'conf/test_targeted.config'
+    }
+    test_targeted {
+        includeConfig 'conf/test_targeted.config'
+    }
+    test_full {
+        includeConfig 'conf/test_targeted_full.config'
+    }
+    test_targeted_full {
+        includeConfig 'conf/test_targeted_full.config'
+    }
+    test_umis {
+        includeConfig 'conf/test_umis.config'
+    }
+    test_screening_full {
+        includeConfig 'conf/test_screening_full.config'
+    }
+    test_screening {
+        includeConfig 'conf/test_screening.config'
+    }
+    test_screening_paired {
+        includeConfig 'conf/test_screening_paired.config'
+    }
+    test_screening_rra {
+        includeConfig 'conf/test_screening_rra.config'
+    }
+    test_screening_count_table {
+        includeConfig 'conf/test_screening_count_table.config'
+    }
 }
 
 // Load nf-core custom profiles from different Institutions
 includeConfig !System.getenv('NXF_OFFLINE') && params.custom_config_base ? "${params.custom_config_base}/nfcore_custom.config" : "/dev/null"
 
+
+// Nextflow plugins
+plugins {
```

Comment:
there is another `plugins` scope, they should be unified because having two different scopes might cause problems